test/contrib/ospf.uts

```diff
@@ -47,3 +47,19 @@ assert a.answers(b)
 pkt = Ether(dst="01:00:5e:00:00:05", src="ca:11:09:b3:00:1c")/IPv6(dst="::1", src="fe80::160c:12aa:fe7e:cd28")/OSPFv3_Hdr(src="75.1.3.1")/\
       OSPFv3_Hello(options=0x12, router="10.75.0.254", backup="10.75.0.1", neighbors=["75.1.0.1"])
 assert raw(pkt) == b'\x01\x00^\x00\x00\x05\xca\x11\t\xb3\x00\x1c\x86\xdd`\x00\x00\x00\x00(Y@\xfe\x80\x00\x00\x00\x00\x00\x00\x16\x0c\x12\xaa\xfe~\xcd(\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x01\x03\x01\x00(K\x01\x03\x01\x00\x00\x00\x00Y\x98\x00\x00\x00\x00\x00\x00\x01\x00\x00\x12\x00\n\x00(\nK\x00\xfe\nK\x00\x01K\x01\x00\x01'
+
+= OSPFv2 Opaque lsa
```

Comment:
Could you also add basic building unit tests for the three new classes?

Such as:
```
>>> r = b'\x00\x01\x00\x0b\xc0\xa8\x00\x00\x02\x02\x02\x02\x80\x00\x00\x01d\xe9\x00\x18data'
>>> p = OSPF_AS_Scope_Opaque_LSA(r)
>>> assert p.type == 11 and p.chksum == 0x64e9
```

Reply:
Done. I didn't use the checksum but I added a length test.